setup.py

```diff
@@ -36,6 +37,7 @@ def run(self):
         if os.path.isfile(COMPILED_CONTRACTS) is False:
             try:
                 self.run_command('build')
+                pass
```

Comment:
what is this doing here @pcppcp? Did you mean to write `return`? It is not the same thing.

Reply:
`pass` in try block shouldn't be there of course. 
The reason for catching the `SystemExit` is that upon completion, `run_command(build)` internally calls `sys.exit` and the whole thing stops.